style.css

```diff
@@ -205,6 +248,9 @@ nav {
   background-position: center;
   background-repeat: no-repeat;
   height: 386;
+  display: flex;
+  flex-direction: column;
+  justify-content: end;
 }
```

Comment:
- [ ] I kindly suggest that only when a project is hovered on does a See Project button appears to the bottom of the image and takes the whole image width 👍🏿. 
- [ ] Also, when a project is hovered on with the mouse the description and dark opacity disappear
 
- [ ] This video below shows the expected behavior of your projects on hover.
https://www.loom.com/share/66329832a272445b989fa32dac2e5796

On hover your project should look like the card that has the arrow in the screenshot below, please achieve this functionality for all cards in this section 👍🏿

| Text | Images |
| ------------- | ------------- |
| Your Design | <img width="763" alt="image" src="https://user-images.githubusercontent.com/52934128/230754412-7aeeef81-e8bd-4cde-84da-df35c80bcffb.png"> |
| Figma Design  | <img width="588" alt="image" src="https://user-images.githubusercontent.com/52934128/230754463-a64e12d6-bf18-4160-bc9c-ecdbacf3839d.png">  |